### DISCUSSION TITLE
How to modify Mach-O files?

### DISCUSSION THREAD

**Discussion by Fi5t** [blacktop/go-macho#68] [Q&A] [answered]
Hello! I used to use LIEF to work with Mach-O files, but now I want to try this library for my tasks. Unfortunately I didn't find any examples how to modify different parts of the file. Is it even possible using this library?

**Comment by blacktop** [comment]
here is an example of me doing it in `ipsw` https://blacktop.github.io/ipsw/docs/guides/macho#macho-patch

**Comment by blacktop** [comment, marked as answer]
https://github.com/blacktop/ipsw/blob/master/cmd/ipsw/cmd/macho/macho_patch_add.go

**Comment by Fi5t** [comment]
@blacktop oh, thank you! It looks promising. I'll dig into it later.
